decentraland/screens/ListLandForSaleScreen.js

```diff
@@ -69,14 +69,18 @@ export class ListLandForSaleScreen extends React.Component {
     // Note: Conversion to USD will be implemented on v0.2.0
     const manaPerUsd = 30;
     // Get mana price using string to avoid imprecise rounding (i.e.: 57999.99999999999)
-    const priceMana = `${priceInWei}`.substring(0, `${priceInWei}`.length - 18);
+    // TODO: Use TasitSDK Utils to dealing with BigNumbers (will be implemented on v0.2.0)
+    const priceManaInWei = `${priceInWei}`;
+    const intPriceManaLength = priceManaInWei.length - 18;
+    const intPriceMana = priceManaInWei.substring(0, intPriceManaLength);
+    const priceMana = intPriceMana;
```

Comment:
@marcelomorgado This step doesn't add much, but we can always come back to this next time we touch this code